docs/user/management.asciidoc

```diff
@@ -131,6 +131,8 @@ include::{kib-repo-dir}/management/managing-licenses.asciidoc[]
 
 include::{kib-repo-dir}/management/managing-remote-clusters.asciidoc[]
 
+include::{kib-repo-dir}/management/numeral.asciidoc[]
```

Comment:
Not sure about the location of the doc here.  The top-level items are related to UIs in the Management app.  Can it go  under Index Patters and Fields, so it is closer to Numeric Field Formatters.

If you decided to keep this as a top-level item under Management, move the item up so it is in alphabetical order like the rest of the topics in this section.

Reply:
@gchaps This is the one I think I need help with. The formatting options here are used in multiple places in Management, but also in TSVB and Canvas. It could even be a separate page at the top level, similar to the Limitations page, because it's used across Kibana. Thoughts?

Reply:
@wylieconlon GIven what you said, I think Management is the best place.  I would move it so it appears between License Management and Remote Clusters.

Also, do you think it's worth mentioning in the doc the different places where numeral formatting is used?